packages/react-providers/src/TransactionManagerProvider/provider.tsx

```diff
@@ -114,14 +121,9 @@ export const TransactionManagerProvider = ({
   );
 
   useEffect(() => {
-    if (txQueue.length > 0) {
-      // Check if there is no onGoing item
-      const isOnGoing = txStatus.find((s) => s.status === "ongoing");
-      if (!isOnGoing) {
-        const ext = txQueue[0];
-        // Send first extrinsic item to blockchain
-        sendExtrinsicToChain(ext);
-      }
+    if (txQueue.length) {
+      const started = txStatus.find((s) => s.status === "ongoing");
+      if (started) sendExtrinsicToChain(txQueue[0]);
```

Comment:
This condition should be reverted.

It should  be 

```js
if (!started)
```

Reply:
It was updated, but checking if the status is: broadcasted || inblock. Since ongoing is only the initial state, in this PR, we included more status verification. Previously, only isFinalized was checked, now we verify 3: isFinalized, isInBlock, and isBroadcast